### DISCUSSION TITLE
Alist 的离线下载功能(Aria2)

### DISCUSSION THREAD

**Discussion by hi778899** [AlistGo/alist#1300] [Ideas]
官方可以支持一下Alist的离线下载功能吗？类似于cloudreve之类的，通过aria2上传的网盘。
        离线下载功能由[Aria2](https://aria2.github.io/)驱动，正确配置并启用离线下载后，用户可以创建磁力链、HTTP、种子下载任务，由服务端下载完成后加入到用户文件中。
        对于云策略，离线下载任务完成后，一个列表文件所下载的文件转存到云存储端，在转存前，用户无法下载、管理已完成。用户可以在任务中查看转存任务进度。

**Comment by Limourli-liu** [comment]
用这个就可以完成上述功能：https://p3terx.com/archives/docker-aria2-pro.html
